src/Apps/Auction/Hooks/useCreateTokenAndSubmit.ts

```diff
@@ -80,11 +80,16 @@ export const useCreateTokenAndSubmit = ({
 
     helpers.setSubmitting(true)
 
+    if (!values.address) {
+      helpers.setStatus("SUBMISSION_FAILED")
+      return
+    }
```

Comment:
This caused a bunch of test failures [fixed [here](https://github.com/artsy/force/pull/14898/files#diff-fb8e302bbf531d00bb5320d725e45048f714ccda3f47e775506aedff347c0488R43)] even though I don't think it would be an issue in real life due to form validations. I only did it to satisfy some ts complaints so we could remove it.